routes/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const router = express.Router()
+
+router.get('/', (req, res) => {
+  res.render('landing')
+})
+
+router.post('/login', (req, res) => {
+  res.json({redirectUrl: '/home'})
+})
+
+router.post('/signup', (req, res) => {
+  res.sendStatus(200)
+})
+
+router.get('/home', (req, res) => {
+  res.render('chatroom',{
+    roomList: [
+      {
+        name: 'Dog Party'
+      },
+      {
+        name: 'Grumpy Cat'
+      }
+    ]
```

Comment:
Will this always just render two roomlists? is this part expandable to allow for more rooms in the future?